half-light.js

```diff
@@ -91,6 +91,13 @@
     });
   })
   
+  let script = document.currentScript;
```

Comment:
This doesn't work if this is a module script right?

I think you have to use import.meta somehow?

I wonder if this is also better addressed by exposing an init function with options arg rather than an IIF looking at an attribute?

Reply:
I dont think this can realistically be a module script though unless you want rough FOUC - and if you want that then I would honestly write this completely differently and also have it work with DSD. Though... what's the point of DSD if not to intended to help avoid FOUC in the first place?

Reply:
Yeah you're probably right